api/events/types.go

```diff
@@ -173,8 +173,21 @@ func ConvertRange(chain *chain.Chain, r *Range) (*logdb.Range, error) {
 			To:   toHeader.Number(),
 		}, nil
 	}
+
+	// Units are blocks, locked a max of 28bits in the logdb
+	maxRange := uint32(1<<28 - 1) // Maximum value for 28 bits: 2^28 - 1
```

Comment:
Can we take this value from the constant in sequence.go? So we don't have to deal with scattered values

Reply:
Good point ! Fixed.